### ISSUE TITLE
How do I install in Mac ?

### ISSUE THREAD

**Issue by AbdealiJK** [HouseOps/HouseOps#4]
No instructions found on google. Would be helpful to add.

**Comment by jonatasfreitasv** [comment]
Hi @AbdealiJK, I`m not make package for OSX.

But, for you is very easy, clone this repo and run `npm install && npm run package`

**Comment by AbdealiJK** [comment]
Yep, tried it and it worked. Would be a good idea to have a section in the readme mentioning this as there are specific sections for windows and Linux

**Comment by AbdealiJK** [comment]
:+1: thanks

**Comment by jonatasfreitasv** [comment]
@AbdealiJK 

Test last version in OSX with this packege ?
https://github.com/HouseOps/HouseOps/releases/download/v0.0.3-alpha/HouseOps-0.0.3-mac.zip

Plz!

**Comment by jonatasfreitasv** [comment]
Thx a lot again @AbdealiJK

**Comment by jonatasfreitasv** [comment]
Put http:// in front

**Comment by jonatasfreitasv** [comment]
@AbdealiJK Try new version plz!

**Comment by jonatasfreitasv** [comment]
@AbdealiJK, put http:// in front your localhost:8123...

**Comment by dvska** [comment]
Connection problems,
paste from buffer doesn't work
